code/modules/mod/mod_ai.dm

```diff
@@ -147,7 +150,7 @@
 	if(ismovable(wearer?.loc))
 		return wearer.loc.relaymove(wearer, direction)
 	else if(wearer)
-		ADD_TRAIT(wearer, TRAIT_FORCED_STANDING, MOD_TRAIT)
+		ADD_TRAIT(wearer, TRAIT_FORCED_STANDING, WEAKREF(src))
```

Comment:
Why weakrefs tho? We don't really want datums in the traits list - Maybe `REF(src)` instead?

Reply:
oh i havent coded a lot in a few months i thought weakrefs were the string ones